components/cache/cache_items.rst

```diff
@@ -0,0 +1,117 @@
+.. index::
+   single: Cache Item
+   single: Cache Expiration
+   single: Cache Exceptions
+
+Cache Items
+===========
+
+Cache items are the information units stored in the cache as a key/value pair.
+In the Cache component they are represented by the
+:class:`Symfony\\Component\\Cache\\CacheItem` class.
+
+Cache Item Keys and Values
+--------------------------
+
+The **key** of a cache item is a UTF-8 encoded string which acts as its
+identifier, so it must be unique for each cache pool. You can freely choose the
+keys, but they should only contain letters (A-Z, a-z), numbers (0-9) and the
+``_`` and ``.`` symbols. Other common symbols (such as ``{``, ``}``, ``(``,
+``)``, ``/``, ``\`` and ``@``) are reserved by the PSR-6 standard for future
+uses.
+
+The **value** of a cache item can be any data represented by a type which is
+serializable by PHP, such as basic types (string, integer, float, boolean, null),
+arrays and objects.
+
+Creating Cache Items
+--------------------
+
+Cache items are created with the ``getItem($key)`` method of the cache pool. The
+argument is the key of the item::
+
+    // $cache pool object was created before
+    $numProducts = $cache->getItem('stats.num_products');
+
+Then, use the :method:`Psr\\Cache\\CacheItemInterface::set` method to set
+the data stored in the cache item::
+
+    // storing a simple integer
+    $numProducts->set(4711);
+
+    // storing an array
+    $numProducts->set(array(
+        'category1' => 4711,
+        'category2' => 2387,
+    ));
+
+.. note::
+
+    Creating a cache item and setting its value is not enough to save it in the
+    cache. You must execute the ``save()`` method explicitly on the cache pool.
```

Comment:
@javiereguiluz What do you think about this? Besides this to me this one is ready to be merged.